BricksAndHearts.UnitTests/ControllerTests/Landlord/LandLordControllerUnassignedTests.cs

```diff
@@ -40,11 +40,16 @@ public async void
             A<string>.That.Matches(s => s == msgBody), A<string>.That.Matches(s => s == subject), A<string>.Ignored,
             A<string>.Ignored
         )).WithAnyArguments().DoesNothing();
+
+        // Fixes NullReferenceException when calling TryValidateModel()
+        UnderTest.ObjectValidator = CustomValidator;

```

Comment:
Could move this to the test base class constructor, don't see any reason not to have it available for every test.

Reply:
That's a good idea - done!